### PR TITLE
use common::ToSeconds in local_trajectory_builder and rate_timer.h

### PR DESCRIPTION
**Pull request by spielawa**
use the new overload of ToSeconds (introduced in 
[PR1244](https://github.com/googlecartographer/cartographer/pull/1244/files)) in local_trajectory_builder.

### CODE REVIEW

**Comment by spielawa** [comment]
@CodeArno somehow github doesn't let me add you as a reviewer, but could you please review ;)

File: cartographer/mapping/internal/3d/local_trajectory_builder_3d.cc [hunk 1]
```diff
@@ -289,17 +285,13 @@ LocalTrajectoryBuilder3D::AddAccumulatedRangeData(
...
     double insert_into_submap_fraction =
-        std::chrono::duration_cast<std::chrono::duration<double>>(
-            insert_into_submap_duration)
-            .count() /
+        common::ToSeconds(insert_into_submap_duration) /
         common::ToSeconds(sensor_duration.value());
     kLocalSlamInsertIntoSubmapFraction->Set(insert_into_submap_fraction);
   }
 
   auto duration = std::chrono::steady_clock::now() - accumulation_started_;
```
Comment (wohe) [inline cartographer/mapping/internal/3d/local_trajectory_builder_3d.cc]: `const auto accumulation_duration`?

File: cartographer/mapping/internal/3d/local_trajectory_builder_3d.cc [hunk 2]
```diff
@@ -289,17 +285,13 @@ LocalTrajectoryBuilder3D::AddAccumulatedRangeData(
       insert_into_submap_stop - insert_into_submap_start;
   if (sensor_duration.has_value()) {
     double insert_into_submap_fraction =
```
Comment (wohe) [inline cartographer/mapping/internal/3d/local_trajectory_builder_3d.cc]: `const`?

File: cartographer/mapping/internal/2d/local_trajectory_builder_2d.cc [hunk 3]
```diff
@@ -241,9 +241,7 @@ LocalTrajectoryBuilder2D::AddAccumulatedRangeData(
       time, range_data_in_local, filtered_gravity_aligned_point_cloud,
       pose_estimate, gravity_alignment.rotation());
   auto duration = std::chrono::steady_clock::now() - accumulation_started_;
```
Comment (wohe) [inline cartographer/mapping/internal/2d/local_trajectory_builder_2d.cc]: `const auto accumulation_duration`?

Comment (spielawa) [reply cartographer/mapping/internal/2d/local_trajectory_builder_2d.cc]: done

File: cartographer/mapping/internal/3d/local_trajectory_builder_3d.cc (hunk 2 above)
Comment (spielawa) [reply cartographer/mapping/internal/3d/local_trajectory_builder_3d.cc]: done

File: cartographer/mapping/internal/3d/local_trajectory_builder_3d.cc (hunk 1 above)
Comment (spielawa) [reply cartographer/mapping/internal/3d/local_trajectory_builder_3d.cc]: done

**Comment by wohe** [comment]
@wally-the-cartographer merge

**Comment by wohe** [comment]
@spielawa Seems this has merge conflict. Can you merge master?

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user wohe. Merge queue now has a length of 1.

**Comment by wally-the-cartographer** [comment]
Error: exit status 1

Log:
~~~
=> handle work item: PR 1248 on googlecartographer/cartographer.
 => Switching to branch master.
 $ git reset --hard HEAD [data/cartographer]
HEAD is now at afa3ba53 Fix the name of real_time_correlative metric in 2D. (#1247)
 $ git clean -fd [data/cartographer]
 $ git checkout master [data/cartographer]
Already on 'master'
Your branch is up to date with 'origin/master'.
 $ git pull [data/cartographer]
Already up to date.
Current branch master is up to date.
 => Checking out spielawa/spielawa-use-ToSeconds as local branch pr_1248.
 $ git remote add spielawa git@github.com:spielawa/cartographer.git [data/cartographer]
fatal: remote spielawa already exists.
 $ git fetch spielawa [data/cartographer]
 $ git branch --track pr_1248 spielawa/spielawa-use-ToSeconds [data/cartographer]
Branch 'pr_1248' set up to track remote branch 'spielawa-use-ToSeconds' from 'spielawa'.
 => Switching to branch pr_1248.
 $ git reset --hard HEAD [data/cartographer]
HEAD is now at afa3ba53 Fix the name of real_time_correlative metric in 2D. (#1247)
 $ git clean -fd [data/cartographer]
 $ git checkout pr_1248 [data/cartographer]
Switched to branch 'pr_1248'
Your branch is up to date with 'spielawa/spielawa-use-ToSeconds'.
 $ git pull --force [data/cartographer]
Already up to date.
Current branch pr_1248 is up to date.
 => merge master.
 $ git merge master [data/cartographer]
Auto-merging cartographer/mapping/internal/3d/local_trajectory_builder_3d.cc
CONFLICT (content): Merge conflict in cartographer/mapping/internal/3d/local_trajectory_builder_3d.cc
Auto-merging cartographer/mapping/internal/2d/local_trajectory_builder_2d.cc
Automatic merge failed; fix conflicts and then commit the result.
 => Abandoning branch pr_1248.
 => Switching to branch master.
 $ git reset --hard HEAD [data/cartographer]
HEAD is now at 7c1b1ffc Ran clang-format.
 $ git clean -fd [data/cartographer]
 $ git checkout master [data/cartographer]
Switched to branch 'master'
Your branch is up to date with 'origin/master'.
 $ git branch -D pr_1248 [data/cartographer]
Deleted branch pr_1248 (was 7c1b1ffc).

~~~

**Comment by spielawa** [comment]
Done. @wohe could you please tell Wally again to merge?

**Comment by wohe** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user wohe. Merge queue now has a length of 1.

**Pull request merged**